### ISSUE TITLE
[JOSS review] paper comments

### ISSUE THREAD

**Issue by ygrange** [xuanxu/starmatrix#52]
I read it with interest, some comments, mostly textual advice:

Line 10: "This yields varies" -> "These yields vary"

Line 11-13: I have a few comments about this part. 
* What do you mean by a "simple" stellar population? (Also I wonder if the word "sample" may not be better here)
* "at each time step" -> I am not sure where the time steps came from. Maybe reword it as "This process is repeated several times, where for each iteration the time is advanced by a fixed value/3Myr/whatever is appropriate here.
* You have not defined GCE before (educated guess: Galactic Chemical Element), it would be nice if this is written out at least once somewhere.

Line 17: At the rist of just not knowing the jargon (Google does not really help me here); "neutron rich elements", is that in essence "all heavier elements"? Also I think rewording it as "and one number representing all heavier/neutron-rich (pick what you prefer) elements"

Line 21/22: I wanted to make sure I parsed the Q correctly, so I went to the Ferrini paper out of curiosity. Ferrini states "we use the restitution matrices Qij(m) defined as the fraction of the mass of an element j initially present in a star of mass m that is transformed in element i and ejected", you state "Starmatrix calculates matrices Qij of masses of elements i ejected to the galactic medium as element j". I think those two definitions are eachothers inverse. Is this intentional?

Line 34: "as a open source" -> "as an open source"

Figure 1: "Oxigen" -> "Oxygen"

What I think I would like to read in the paper is something I also mentioned w.r.t. the README: It would be nice to mention what the default configuration values are (I think in the paper you can mention that there are default values and that those are typical values for an XYZ-type situation. 

Also one thing I wonder after reading the paper, but before playing around is: Could I add my own data set (say, I develop my own IMF or supernova yield model; would it be hard to try it out in Starmatrix)? The README claims this is possible, then I would certainly mention it in the paper because I see it as an extra strenth of the tool.

**Comment by xuanxu** [comment]
> Line 10: "This yields varies" -> "These yields vary"

Fixed

> Line 11-13: I have a few comments about this part.
> 
>     * What do you mean by a "simple" stellar population? 

A SSP is a group of stars born at the same time (will have same age) and from the same medium (will have similar metallicity). I think is a common term among the people working on stars/galaxies modelling but I have added that clarification. A couple of links for reference: 
https://www.stsci.edu/hst/instrumentation/reference-data-for-calibration-and-tools/astronomical-catalogs/simple-stellar-population-atlas
https://ned.ipac.caltech.edu/level5/Sept14/Conroy/Conroy2.html
>     * "at each time step" -> I am not sure where the time steps came from. Maybe reword it as "This process is repeated several times, where for each iteration the time is advanced by a fixed value/3Myr/whatever is appropriate here.

The evolution is integrated for the life of the universe, this time step is the step for the integration. Added clarification.

>     * You have not defined GCE before (educated guess: Galactic Chemical Element), it would be nice if this is written out at least once somewhere.

Added! It refers to `galactic chemical evolution`

> 
> Line 17: At the rist of just not knowing the jargon (Google does not really help me here); "neutron rich elements", is that in essence "all heavier elements"?

"neutron rich elements" was short for "neutron rich isotopes from C12 C13 N14 and O16". I have corrected it.

> Line 21/22: I wanted to make sure I parsed the Q correctly, so I went to the Ferrini paper out of curiosity. Ferrini states "we use the restitution matrices Qij(m) defined as the fraction of the mass of an element j initially present in a star of mass m that is transformed in element i and ejected", you state "Starmatrix calculates matrices Qij of masses of elements i ejected to the galactic medium as element j". I think those two definitions are eachothers inverse. Is this intentional?

Right! my definition was written too fast, I fixed it.

> Line 34: "as a open source" -> "as an open source"

Corrected
 
> Figure 1: "Oxigen" -> "Oxygen"

Corrected

> What I think I would like to read in the paper is something I also mentioned w.r.t. the README: It would be nice to mention what the default configuration values are (I think in the paper you can mention that there are default values and that those are typical values for an XYZ-type situation.
> 
> Also one thing I wonder after reading the paper, but before playing around is: Could I add my own data set (say, I develop my own IMF or supernova yield model; would it be hard to try it out in Starmatrix)? The README claims this is possible, then I would certainly mention it in the paper because I see it as an extra strenth of the tool.

Yes you could use your own SN or IMF models, I'll include some example in the documentation on how to do it, then I'll add a mention in the paper.

**Comment by xuanxu** [comment]
I've added a `Default values` section to the docs: 
https://starmatrix.readthedocs.io/en/latest/configuration.html#default-values

**Comment by xuanxu** [comment]
Thanks again @ygrange!
